### PR TITLE
Fix typos in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Amin-Hosseini**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Fix typos in CONTRIBUTING.md
| Type?             | improvement
| Category?         | PM
| BC breaks?        | no
| Deprecations?     | no

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/23109)

### CODE REVIEW

**Review by Progi1984 — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 Contributing to PrestaShop
 --------------------------
 
-PrestaShop is an open-source e-commerce solution. Everyone is welcome and even encouraged to contribute with their own improvements.
+PrestaShop is an Open Source ecommerce solution. Everyone is welcome and even encouraged to contribute with their own improvements.
```
Comment (PierreRambaud) [inline CONTRIBUTING.md]: It's "open source" or "open-source" but not "Open Source". It's not a name like Open Source Foundation or Open Source Initiative

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -21,14 +21,14 @@ Contributors should follow the following process:
 7. Create a pull request for your changes on the PrestaShop project. Be sure to follow [the contribution guidelines][2] in your pull request. If you need help to make a pull request, read the [Github help page about creating pull requests][3].
 8. Wait for one of the core developers either to include your change in the codebase, or to comment on possible improvements you should make to your code.
 
-That's it: you have contributed to this open source project! Congratulations!
+That's it: you have contributed to this Open Source project! Congratulations!
```
Comment (PierreRambaud) [inline CONTRIBUTING.md]: same as above

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
Few comments

File: CONTRIBUTING.md (hunk 1 above)
Comment (Amin-Hosseini) [reply CONTRIBUTING.md]: Yeah @PierreRambaud :blush: Your right

**Comment by Amin-Hosseini** [comment]
@PierreRambaud Changes applied :+1:

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
 Contributing to PrestaShop
 --------------------------
 
-PrestaShop is an open-source e-commerce solution. Everyone is welcome and even encouraged to contribute with their own improvements.
+PrestaShop is an open source ecommerce solution. Everyone is welcome and even encouraged to contribute with their own improvements.
```
Comment (PierreRambaud) [inline CONTRIBUTING.md]: It's the same for e-commerce. I see no reason to change the working here.
Could you please revert your changes.
The only thing acceptable in this pull request is what you did with links :)

Comment (Amin-Hosseini) [reply CONTRIBUTING.md]: I reverted my changes :+1: 
Thank you @PierreRambaud

**Review by PierreRambaud — APPROVED** [review]
I rebase your PR ;)

**Review by matks — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @Amin-Hosseini

**Pull request merged**